### ISSUE TITLE
Page<T, T2> support

### ISSUE THREAD

**Issue by bfcamara** [CollaboratingPlatypus/PetaPoco#85]
I am trying to do this

            var asset = db.Page<Asset>(1, 10, PetaPoco.Sql.Builder
                .Select()
                .From("Asset")
                .InnerJoin("AssetType").On("AssetTypeId = AssetType.Id"));

When I run the query I get 

The column 'Id' was specified multiple times for 'peta_paged'.

It seems there is no support for

            var asset = db.Page<Asset, assetType>(1, 10, PetaPoco.Sql.Builder
                .Select()
                .From("Asset")
                .InnerJoin("AssetType").On("AssetTypeId = AssetType.Id"));

Basically, I want pagonation with eager loading of my POCOs relations. Any help?

Thanks in advance
